asllib/ASTUtils.ml

```diff
@@ -49,9 +49,13 @@ module IMap = struct
 end
 
 let dummy_pos = Lexing.dummy_pos
+let dummy_version = V1
 let desc v = v.desc
-let annotated desc pos_start pos_end = { desc; pos_start; pos_end }
-let add_dummy_pos desc = annotated desc dummy_pos dummy_pos
+
+let annotated desc pos_start pos_end version =
+  { desc; pos_start; pos_end; version }
+
+let add_dummy_pos desc = annotated desc dummy_pos dummy_pos dummy_version
```

Comment:
rename `annotate_dummy_pos` to something like `dummy_annotation`, since annotations are not just about source position now.